xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -634,10 +651,20 @@
         <Parameter Name="item" Type="System.Text.RegularExpressions.Capture" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="item">The object to remove from the collection.</param>
+        <summary>Removes the first occurrence of a specific object from the collection.</summary>
+        <exception cref="T:System.NotSupportedException">The collection is read-only.</exception>
+        <returns>
+          <see langword="true" /> if <paramref name="item" /> was successfully removed from the collection; otherwise, <see langword="false" />. This method also returns <see langword="false" /> if <paramref name="item" /> is not found in the original collection.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This method is not supported. This is a read-only collection.
+
+          ]]></format>
+        </remarks>
```

Comment:
To keep all APIs consistent, let's not mention this text in the remarks, and instead make sure summaries and return values mention that the API throws in all cases because the collection is read-only.

```suggestion
        <remarks>To be added.</remarks>
```

Reply:
Done. Removed these comments everywhere